docs/openapi.json

```diff
@@ -665,6 +667,9 @@
                   "access_token": {
                     "type": "string"
                   },
+                  "cpu_usage": {
+                    "type": "string"
```

Comment:
Maybe it doesn't belong to this PR as I see it's the same for `disk_usage` but,  shouldn't this be an integer?

Reply:
Yes! I see it as something to deal with while doing https://github.com/reanahub/reana-server/issues/289, what do you think?